This patch fixes a crash when the "Refresh" button in the "Tracker Details"
tab is pushed, due to the use of an uninitialized GValue.

https://bugzilla.redhat.com/show_bug.cgi?id=542502
https://bugzilla.redhat.com/show_bug.cgi?id=572806
https://bugzilla.redhat.com/show_bug.cgi?id=593734

--- GTorrentViewer-0.2b/src/mainwindow.c.orig	2004-10-26 05:37:58.000000000 +0100
+++ GTorrentViewer-0.2b/src/mainwindow.c	2010-06-25 14:43:22.391786427 +0100
@@ -744,7 +744,7 @@
 {
   GtkListStore *list;
   GtkTreeIter iter;
-  GValue itemvalue;
+  GValue itemvalue = {0};
   gchar *tracker;
   GError *err;
   
